resources/styles/components/task-plan/builder.less

```diff
@@ -0,0 +1,20 @@
+.assignment {
+
+  label.all-periods {
+    .form-control-wrapper .floating-label;
+    opacity: 1;
+    top: 0px;
+    left: 35px;
+  }
+  .periods-listing {
+    .table;
+    .table-striped;
+    .table-bordered;
+    .form-control { background-image: inherit; }
+  }
+
+  .instructions {
+    color: @tutor-neutral;
+    #fonts .sans(1.1rem, 1.1rem);
```

Comment:
Can you include `>` to make the markup the same as other places where this is used?